test/commands/users/list.test.ts

```diff
@@ -1,17 +1,29 @@
-import {expect, test} from '@oclif/test'
+import { expect, test } from "@oclif/test";
+import { hasCosmosConnection } from "../../../src/utils/azure";
 
-describe('users:list', () => {
-  test
-    .stdout()
-    .command(['users:list'])
-    .it('runs hello', ctx => {
-      expect(ctx.stdout).to.contain('hello world')
-    })
-
-  test
-    .stdout()
-    .command(['users:list', '--name', 'jeff'])
-    .it('runs hello --name jeff', ctx => {
-      expect(ctx.stdout).to.contain('hello jeff')
-    })
-})
+describe("list all profiles", () => {
+  const isCosmosConnectionAvailable = hasCosmosConnection(
+    "agid-rg-test",
+    "agid-cosmosdb-test"
+  );
+  /**
+   * if host has the az client installed and valid cosmos credentials we test command first output line matches
+   * the header row (fiscalCode column name). Otherwise the test expects for an error with 2 as exit status
+   */
+  if (isCosmosConnectionAvailable) {
```

Comment:
these should be two different tests - you can follow [this pattern](https://github.com/facebook/jest/issues/7245#issuecomment-440774078) to skip tests that require `az`